mypy/stubgenc.py

```diff
@@ -257,10 +314,12 @@ def generate_c_function_stub(
     module: ModuleType,
     name: str,
     obj: object,
+    known_modules: list[str],
```

Comment:
Maybe make the argument keyword-only here.

Reply:
Done